kusto-language-server/client/src/extension.ts

```diff
@@ -105,6 +105,16 @@ commands.registerCommand('kuskus.loadSymbols', async () => {
 			return;
 		}
 
+		const authId = await window.showInputBox({
+			ignoreFocusOut: true,
+			placeHolder: 'common',
```

Comment:
What are some other common values for authority / tenant id? I'm not familiar with it. I suppose my use case is the "common" one.

Reply:
I believe they're specific to your azure AD, for example if i'm using Company A resources, then i would use their tenant/authority id XXXX-XXXX-XXXX and for Company B I would use their tenant id YYYY-YYYY-YYYYY.